security/remember_me.rst

```diff
@@ -261,3 +257,116 @@ your controller using annotations::
 
 For more information on securing services or methods in this way,
 see :doc:`/security/securing_services`.
+
+.. _token_in_database:
+
+Storing Remember Me Tokens in the Database
+------------------------------------------
+
+By default, tokens are stored in a cookie. You can choose to store the token in a database,
+to not have a (hashed) version of the password in a cookie.
```

Comment:
why (hashed) with parenthesis? 

also, is it really the password? or like an hash?

Reply:
> why (hashed) with parenthesis?

This was just copied from the original description, I'm not sure why the parenthesis is needed

> also, is it really the password? or like an hash?

It's a hashed version of the password